src/util/dom.ts

```diff
@@ -84,22 +90,35 @@ export class DOM {
         }, 0);
     }
 
-    public static mousePos(el: HTMLElement, e: MouseEvent | Touch) {
-        const rect = el.getBoundingClientRect();
+    private static getScale(element: HTMLElement): ScaleReturnValue {
+        const rect = element.getBoundingClientRect();
+        return {
+            x: rect.width / element.offsetWidth || 1,
```

Comment:
```suggestion
            x: (rect.width / element.offsetWidth) || 1,
```

(for order of operations clarity)